packages/main/src/Link.js

```diff
@@ -364,11 +388,12 @@ class Link extends UI5Element {
 			return;
 		}
 
-		event.preventDefault();
+		this._onclick(event);
+
+		if (this.href && !event.defaultPrevented) {
+			const fakeEvent = new MouseEvent("click");
```

Comment:
Is fakeEvent the best name for this variable?

Reply:
If you have any suggestion, but we create event that will be dispatched through link to simulate click from the link to achieve the default behaviour of link click.

Reply:
customEvent for example